lib/sync/installation.js

```diff
@@ -102,14 +102,19 @@ module.exports.processInstallation = (app, queues) => {
       return processor(github, repository, cursor, perPage)
     }
 
+    const processJobGitHubError = (err) => {
+      if (!String(err).includes('MAX_NODE_LIMIT_EXCEEDED') ||
```

Comment:
This code behaves the way we want but is dense<sup>1</sup>. If I were to explain this to someone, I'd say it something like this: "swallow ignored errors, otherwise throw"

Can we make the code say that too? Here's an example refactoring:

```javascript
const processJobGitHubError = (err) => {
  const ignoredErrors = [
    'MAX_NODE_LIMIT_EXCEEDED',
    'NOT_FOUND'
  ]

  const isIgnoredError = ignoredErrors.find(ignoredError => String(err).includes(ignoredError))

  if (!isIgnoredError) {
    throw err
  }
}
```

---

1: "Dense" is ambiguous. Let me clarify what I mean. When a single statement/expression contains more than one "thing", the density goes up. Lines became hard to understand at 2-3 things. For this `if` statement, I count 3 things: how to check if an error is ignored, what errors are ignored, what we do when an error is ignored. The `||` and two `!` make it denser yet.